packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.constants.js

```diff
@@ -0,0 +1,18 @@
+export const ASIA_FORMAT = ['SG', 'ASIA', 'AU'];
+export const FRENCH_FORMAT = ['CZ', 'ES', 'FR', 'GB', 'IE', 'IT', 'LT', 'MA', 'NL', 'PL', 'PT', 'TN'];
+export const GERMAN_FORMAT = ['DE', 'FI', 'SN'];
+export const US_FORMAT = ['CA', 'WE', 'WS', 'QC', 'US'];
```

Comment:
I don't know the purpose yet, but won't something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl could help ?

Reply:
No, it's more complex than that unfortunately